README.md

```diff
@@ -116,13 +116,13 @@ However, some features may not be possible/present in older Python versions due
 
 ## Contributing
 
-Feel free to open issues, bugreports or pull requests.  
```

Comment:
:x: 
the two white spaces were there for a reason. they are line breaks in markdown.
please revert!